test/test_open.py

```diff
@@ -12,6 +13,16 @@
 FILENAME = Path('one')
 
 
+# Windows OS helper for checking admin privileges
+def is_windows_admin():
+    """Check if the script is running in a terminal with admin privileges on Windows"""
+    if os.name == 'nt':
+        try:
+            return ctypes.windll.shell32.IsUserAnAdmin()
+        except:
```

Comment:
Bare `except` is a bad idea because you catch things that aren't actually `Exception` - things like `OutOfMemoryError` or `KeyboardInterrupt`.

I think the CI (that I just enabled) will catch it, but if not, here's my general purpose lint that I use: https://github.com/rec/dotfiles/blob/master/bin/run-test

Reply:
Thanks for the review!
I'll get started on implementing your recommendations.

Reply:
Would `except Exception as e` work just fine for this case?

Reply:
Yes, that would work much better!

Linting would reveal a tiny defect: you don't use the new variable `e`.

Best is `exception Exception:`.

**On linting**

The few years have seen a significant change in how people write Python. More and more, projects simply require you to run [`ruff`](https://github.com/astral-sh/ruff) with all the features turned on, on your code.

I think this is great - the code gets standardized, common errors and defects are avoided, and `ruff` is blazingly fast.

You should get into the habit of using `ruff` with a lot of stuff turned on, and also `mypy`, even on trivial code, because it catches so many bugs.